bin/alluxio

```diff
@@ -46,77 +46,92 @@ function bootstrapConf {
     echo ${usage}
     exit 1
   fi
+  local master=${1}
   local ufs=${2:-local}
 
   ALLUXIO_CONF_DIR="${BIN}/../conf"
+  local src="${ALLUXIO_CONF_DIR}/alluxio-env.sh.template"
   local dst="${ALLUXIO_CONF_DIR}/alluxio-env.sh"
+
   if [[ -e ${dst} ]]; then
     echo "${dst} already exists"
     return
   fi
 
+  if [[ ! -e ${src} ]]; then
+    echo "Cannot find ${src}"
+    exit 1
+  fi
+
   if [[ $(uname -s) == Darwin ]]; then
-    # osx sed wants an argument to -i, use .bootstrap.bk
-    ALLUXIO_SED="sed -i .bootstrap.bk"
+    # Assuming Mac OS X
     TOTAL_MEM=$(sysctl hw.memsize | cut -d ' ' -f2)
-    TOTAL_MEM=$[TOTAL_MEM / 1024]
+    TOTAL_MEM=$[TOTAL_MEM / 1024 / 1024]
```

Comment:
the reason for this change is that, originally the unit for total mem is in "KB", which I feel is hard to read when the number if large. So I changed the unit to be "MB"